app/js/states/inventory/inventory.jade

```diff
@@ -49,10 +49,10 @@
                   ng-change='selectAll(); checkboxes.checkboxChecked(checkboxes.checked, getSelectableSystems())')
 
 
-              th.fit-content.min.sortable(ng-class="predicate | sortClass:'system_type_id':reverse", ng-click="sort('system_type_id')", translate) System Type
-              th.sortable(ng-class="predicate | sortClass:'toString':reverse", ng-click="sort('toString')", translate) System Name
-              th.fit-content.min.sortable(ng-class="predicate | sortClass:'last_check_in':!reverse", ng-click="sort('last_check_in')", translate) Last Check In
-              th.min.sortable(ng-class="predicate | sortClass:'report_count':reverse", ng-click="sort('report_count')", translate) Status
+              th.fit-content.min.sortable(ng-class="sorter.predicate | sortClass:'system_type_id':sorter.reverse", ng-click="sort('system_type_id')", translate) System Type
+              th.sortable(ng-class="sorter.predicate | sortClass:'toString':sorter.reverse", ng-click="sort('toString')", translate) System Name
+              th.fit-content.min.sortable(ng-class="sorter.predicate | sortClass:'last_check_in':sorter.reverse", ng-click="sort('last_check_in')", translate) Last Check In
+              th.min.sortable(ng-class="sorter.predicate | sortClass:'report_count':sorter.reverse", ng-click="sort('report_count')", translate) Status
```

Comment:
we should be calling sorter.sort in the ng-click which will set reverse, and predicate. Then the callback is called which in this case is set to order(). Order should then reset the scope and call getData().